rts/Lua/LuaSyncedCtrl.h

```diff
@@ -27,6 +27,7 @@ class LuaSyncedCtrl
 		inline static int inDestroyFeature = 0;
 		inline static int inGiveOrder = 0;
 		inline static int inTransferUnit = 0;
+		inline static int inTransferUnitLimit = 0;
```

Comment:
Transferring unit limit doesn't produce an event so it doesn't need a recursion check.